### PR TITLE
Refactor contact list

### PR DESCRIPTION
**Pull request by Djon-Latii**

### CODE REVIEW

File: src/models/myContact.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export var ContactModel =  {
```
Comment (AlexBychkov) [inline src/models/myContact.js]: it's better to call the file the same as class -  ContactModel

File: src/pages/contact/Contact.js [hunk 2]
```diff
@@ -11,10 +11,27 @@ import {
   ListItemSecondaryAction,
 } from '@material-ui/core';
 import DeleteIcon from '@material-ui/icons/Delete';
+import AddIcon from '@material-ui/icons/Add';
 
 class Contact extends Component {
   render() {
     let user = this.props.contactData;
+    let btnAddDel
```
Comment (AlexBychkov) [inline src/pages/contact/Contact.js]: maybe it's better to initialize this btn with any value, empty for example.
Or maybe just make a condition differently, like this:

 btnAddDel =  <ListItemSecondaryAction onClick={() => user.friend ? this.props.deleteContact(user.id) : this.props.addContact(user.id)}>
      <IconButton edge="end" aria-label="delete">
        (user.friend) ? <DeleteIcon /> : <AddIcon /> <- this lne should be tested, maybe syntax is incorrect
      </IconButton>
    </ListItemSecondaryAction>

File: src/pages/contact/modalAddContact/ModalAddContact.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+  componentDidMount() {
+    this.loadAllContacts();
+  }
+
+  render() {
+    return (
+      <Modal
+        open={this.props.isOpen}
+        onClose={() => this.props.toggleModal()}
```
Comment (AlexBychkov) [inline src/pages/contact/modalAddContact/ModalAddContact.js]: for methods without parameters you could skip creating anonymous method and just do like that -  onClose={this.props.toggleModal} - this should work

**Review by AlexBychkov — CHANGES_REQUESTED** [review]
some minor comments. should be quick to fix

**Review by AlexBychkov — APPROVED** [review]

**Review by Nicelx — APPROVED** [review]

**Pull request merged**